.github/workflows/build_compiler.yml

```
@@ -0,0 +1,43 @@
+name: Build-Thrift-Compiler
+on: [push]
```

This should build for PRs, too.

```suggestion
on:
  push:
    branches: [ '*' ]
  pull_request:
    branches: [ '*' ]
```